### ISSUE TITLE
[BUG] - [DND5e] - Items dragging problems

### ISSUE THREAD

**Issue by escixi** [fantasycalendar/FoundryVTT-ItemPiles#648]
**Are you on the latest version of Item Piles?**

Yes

**Have you tried to reset Item Piles' system settings?**

Yes

**Describe the bug**

Items dragged from a token will not go to an item pile or container and items dragged from an item pile or container to a token gets dup. Vault still works as intended.

**To Reproduce**

Steps to reproduce the behavior: As described, no setting changes, new world.

**Expected behavior**

Items dragged from a token will go to an item pile or container and items dragged from an item pile or container to a token do not get duplicated.

**Screenshots**
Can't drag item into a pile
![Can't drag into a pile](https://github.com/user-attachments/assets/46931d2b-1aae-4392-8f18-a6edc57cb08a)
If drag from a pile, item got dup
![If drag from a pile, item dup](https://github.com/user-attachments/assets/ef558c2c-a55f-41d9-ae43-6a6dbc14c27a)

**Setup:**

- Foundry Version: 12.330

- System Version: DND5e 3.3.1

- Item Piles Version: 3.1.1

**Active modules:**

- Item Piles
- libwrapper
- socketlib

**Comment by Haxxer** [comment]
I cannot overwrite how system sheets handle dropped data. I can only control how dropping items are handled in sheet I have created.

The intended use case is to not open it like a GM, but to select the token next to the item pile, then shift double click on the item pile and use the "Take" button in the item piles UI. This is described in the wiki:
https://fantasycomputer.works/FoundryVTT-ItemPiles/#/creating-item-piles?id=inspecting-pile-as-character

**Comment by escixi** [comment]
I'm still confused about not being able to add items to an existing pile or container. Creating a new pile works, but dragging items on top of an existing pile or the interface seems to ask about adding item to the pile, but then nothing happens. Did I get something wrong? Or do I have to do something else? I'm so sorry, I'm just so confused.

I tried to do it like on the wiki.
![image](https://github.com/user-attachments/assets/de35cd8a-766a-4a01-964a-0cb5ac450b3e)

Here is the error that came up when I tried to drop an item on the pile.
![image](https://github.com/user-attachments/assets/e358bb33-d1dc-4ad9-9094-933ebd370722)
